Competition_Task/Competition_Task/Tests/CertificationsTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿using Competition_Task.Pages;
+using Competition_Task.TestData;
+using Competition_Task.Utilities;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Competition_Task.Tests
+{
+    [TestFixture]
+    
+    public class CertificationsTests:BaseClass
+    {       
+
+            LoginPage LoginPageObj = new LoginPage();
+            HomePage HomePageObj = new HomePage();
+            CertificationsTab CertificationsTabObj = new CertificationsTab();
+
+
+            public static string certificatedata = File.ReadAllText(@"A:\Industry Connect\MarsCompetition_Task\Competition_Task\Competition_Task\TestData\Certifications.json");
+            public static JArray certificationvalues = JArray.Parse(certificatedata);
+
+
+            IList<Certification> certifications = certificationvalues.Select(p => new Certification
+            {
+                CertificateName = (string)p["CertificateName"],
+                CertifiedFrom = (string)p["CertifiedFrom"],
+                Year = (string)p["Year"]
+            }).ToList();
+
+            [SetUp]
+            public void SetUpCertificateTest()
+            {
+                   Initialize();
+
+                LoginPageObj.LogInActions();
+                Thread.Sleep(2000);
+                driver.Navigate().Refresh();
+                HomePageObj.SelectCertficationTab();
+                CertificationsTabObj.DeleteExistingCertficationRecords();
+
+
+            }
+
+            [Test, Order(1), Description("Adding a new Certificate Record")]
+            public void AddaNewCertificateRecordwithValidData()
+            {
+
```

Comment:
can you please create individual json file for each test, in theory all test data files should be unique per test and should not be shared, please fix this in all test

Reply:
Created separate files 
![image](https://github.com/Shilpii01/MarsCompetition_Task/assets/145233624/11f642ef-d89b-41b8-b386-9a0ef2c87357)